kroxylicious-filters/kroxylicious-record-encryption/src/main/java/io/kroxylicious/filter/encryption/config/RecordEncryptionConfig.java

```diff
@@ -49,6 +55,10 @@ public DekManagerConfig dekManager() {
 
     @Nullable
     private Integer getExperimentalInt(String property) {
+        return getExperimentalIntElse(property, null);
+    }
+
+    private Integer getExperimentalIntElse(String property, Integer defaultValue) {
```

Comment:
nit: the general pattern is 
```suggestion
    private Integer getExperimentalIntOrElse(String property, Integer defaultValue) {
```